app/assets/javascripts/application.js

```diff
@@ -13,6 +13,8 @@
 //= require jquery_ujs
 //= require dsds
 //= require date_chooser
+//= require show_hide
+//= require disable_checker
```

Comment:
I was more thinking in the `page_initialise` file where they're used. But yeah, no biggie.